ballerina/os.bal

```diff
@@ -58,3 +58,58 @@ public isolated function getUserHome() returns string = @java:Method {
     name: "getUserHome",
     'class: "io.ballerina.stdlib.os.nativeimpl.GetUserHome"
 } external;
+
+# Sets the value of the environment variable named by the key. 
+# Note that the parameter key cannot be an empty string or "==" sign.
+# ```ballerina
+# os:Error? err = os:setEnv("BALCONFIGFILE", "/path/to/Config.toml");
+# ```
+#
+# + key - Key of the environment variable
+# + value - Value of the environment variable
+# + return - An `os:Error` if setting the environment variable fails, () otherwise
+public isolated function setEnv(string key, string value) returns Error? {
+    if key == "" {
+        return error Error("The parameter key cannot be an empty string");
+    } else if key == "=="  {
+        return error Error("The parameter key cannot be == sign");
+    } else {
+        return setEnvExtern(key, value, false);
+    }
+}
+
+# Removes a single environment variable from the system if it exists.
+# Note that the parameter key cannot be an empty string.
+# ```ballerina
+# os:Error? err = os:unsetEnv("BALCONFIGFILE");
+# ```
+#
+# + key - Key of the environment variable
+# + return - An `os:Error` if unsetting the environment variable fails, () otherwise
+public isolated function unsetEnv(string key) returns Error? {
+    if key == "" {
+        return error Error("The parameter key cannot be an empty string");
+    } else {
+        return setEnvExtern(key, "", true);
+    }
```

Comment:
Don't we need `==` check in unset function

Reply:
The `==` check is needed only for `setEnv` function. In `unsetEnv` function this will be treated as another key that does not exist in the environment variable map and hence does not return an error.

Reply:
okay